Assets/tempScriptMove.cs.meta

```diff
@@ -0,0 +1,11 @@
+fileFormatVersion: 2
+guid: e8736de870420a149bcf214ae4065bd0
+MonoImporter:
+  externalObjects: {}
+  serializedVersion: 2
+  defaultReferences: []
+  executionOrder: 0
+  icon: {instanceID: 0}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant: 
```

Comment:
All files after this until ProjectSettings/QualitySettings.asset should be deleted.